### PR TITLE
Add colors to pill bottles and patch packs

### PR DESCRIPTION
**Pull request by Arthri**
## What Does This PR Do
- Pill bottle boxes and patch pack boxes now spawn with colored contents
- Changed ChemMaster container colors
  - Added indigo, violet, and purple
  - Removed blue
  - Replaced red with light red
- Made patch pack color overlay lighter

## Why It's Good For The Game
Great for marking different meds, such as emergency meds in red pill bottles

Pure red and blue on patch packs/pill bottles is an eye sore, so I've made red lighter and removed blue(light blue replacing it)

Colored patch packs are now more saturated, making them easier to discern

## Images of changes
Pill Bottles
Patch Packs

## Testing
Spawned in, opened a few boxes and checked that they have color.

## Changelog
:cl: cdui
add: ChemMaster can now paint pill bottles and patch packs indigo, violet, and purple
del: ChemMaster can no longer paint pill bottles and patch packs pure blue. Light blue is still an option, however
tweak: ChemMaster's "Red" color for pill bottles and patch packs is now lighter
tweak: Colored patch packs are now lighter
tweak: Pill bottle boxes and patch pack boxes now spawn with their contents colored
/:cl:

### CODE REVIEW

**Comment by Scribble-Sheep** [comment]
Interesting. Is there any way to change it for crafted/empty pillbottles, or is it entirely at the mercy of the roundstart contents? How is the maints floor pills bottle handled?

**Comment by CinnamonSnowball** [comment]
> Some of the colors used here are exclusive to the boxes

I'm fairly sure ChemMaster UI should have enough colors for this? We shouldn't really be creating unreplicable items for no reason, if avoidable

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -433,14 +433,20 @@
 	new /obj/item/reagent_containers/hypospray/autoinjector/stimpack(src)
 	new /obj/item/reagent_containers/hypospray/autoinjector/stimpack(src)
 
+/proc/apply_wrapper_color(obj/item/storage/pill_bottle/bottle, i)
+	var/static/list/colors = list(COLOR_RED, COLOR_ORANGE, COLOR_YELLOW, COLOR_GREEN, COLOR_CYAN_BLUE, COLOR_VIOLET, COLOR_PURPLE)
+	bottle.wrapper_color = colors[(i - 1) % length(colors) + 1]
+	bottle.apply_wrap()
```
Comment (Sirryan2002) [inline code/game/objects/items/weapons/storage/boxes.dm:439]: could this proc be attached to pill bottles instead of as a global proc?

**Review by Sirryan2002 — CHANGES_REQUESTED** [review]

**Review by Sirryan2002 — APPROVED** [review]

**Review by DGamerL — COMMENTED** [review]
LGTM - I agree with snowball here, can you update the TGUI so you can pick all colors for the packs/bottles?

**Review by DGamerL — APPROVED** [review]

**Pull request merged**